core/conversion/converters/impl/squeeze.cpp

```diff
@@ -26,7 +26,8 @@ auto squeeze_registrations TORCHTRT_UNUSED = RegisterNodeConversionPatterns().pa
        }
 
        if (selfDim[dim] != 1) {
-         auto out = ctx->AssociateValueAndTensor(n->outputs()[0], self);
+         auto id_layer = ctx->net->addIdentity(*self);

```

Comment:
You could use already existing utility for this https://github.com/pytorch/TensorRT/blob/d3041f553b60cf8639117e77cb9a68675755c307/core/conversion/converters/converter_util.cpp#L124

Reply:
thanks @peri044. BTW, do we still need this test case https://github.com/pytorch/TensorRT/blob/078977c5a24b5149bfb5aa8996982a5b072f4220/tests/core/conversion/converters/test_squeeze.cpp#L29? I'm thinking of deleting it.

Reply:
@peri044  it seems that we should have an explicit name for output tensor if we want to use this util: https://github.com/pytorch/TensorRT/blob/d3041f553b60cf8639117e77cb9a68675755c307/core/conversion/converters/converter_util.cpp#L124. 
I checked around but have no idea what name would be appropriate, any idea?